python/ray/rllib/ddpg/ddpg_evaluator.py

```diff
@@ -0,0 +1,213 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from gym.spaces import Box
+import numpy as np
+import tensorflow as tf
+
+import ray
+from ray.rllib.utils.error import UnsupportedSpaceException
+from ray.rllib.ddpg import models
+from ray.rllib.ddpg.common.wrappers import wrap_ddpg
+from ray.rllib.ddpg.common.schedules import ConstantSchedule, LinearSchedule
+from ray.rllib.optimizers import SampleBatch, PolicyEvaluator
+from ray.rllib.utils.compression import pack
+
+
+def adjust_nstep(n_step, gamma, obs, actions, rewards, new_obs, dones):
```

Comment:
This function looks duplicated from DQN, can we move it to utils.

You can also import it from the `dqn` package for now.

Reply:
adopted the later.